.github/workflows/ci-publish.yml

```diff
@@ -51,16 +57,23 @@ jobs:
           name: dist-artifact
           path: dist/
 
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+
       - name: Install from build
         run: |
           python -m pip install --upgrade pip
-          pip install resqpy[tests] --find-links=dist/
+          pip install poetry-dynamic-versioning
+          poetry install
```

Comment:
Should we use the existing `pip install resqpy --find-links=dist/` here, to ensure we are running the tests against the built package?

This step was originally intended to ensure that the python wheel functions correctly and can be pip-installed by end users.